console-ui/src/pages/ConfigurationManagement/ConfigEditor/NewConfigEditor.js

```diff
@@ -493,11 +518,16 @@ class ConfigEditor extends React.Component {
                   </Checkbox>
                 )}
                 {isBeta && (
-                  <Input.TextArea
-                    aria-label="TextArea"
-                    placeholder="127.0.0.1,127.0.0.2"
-                    value={betaIps}
-                    onChange={betaIps => this.setState({ betaIps })}
+                  <Select
+                    size="medium"
```

Comment:
I think use `tag` mode better, allow input as an option.
doc:https://fusion.design/pc/component/select?themeid=2

Reply:
Yes, It's better to use tag mode if we allow users to input the betaIps which are not in the current Ip list.